docs/source/new_project.rst

```diff
@@ -45,7 +45,7 @@ questions:
    * - *What github organization will your project live under?*
      - This will either be a github organization, or your github username, if you're working outside 
        of an organization. This is used to construct URLs to your project, like
-       ``https://github.com/{{project_organization}}/{{project_name}}``
+       ``git@github.com:{{project_organization}}/{{project_name}}``
```

Comment:
Same here - it's about creating web URLs (e.g. for readme badges) so should stay HTTP.